eng/common/testproxy/scripts/tag-merge/merge-proxy-tags.ps1

```diff
@@ -270,12 +267,22 @@ function Combine-Tags($RemainingTags, $AssetsRepoLocation, $MountDirectory){
     # if we have successfully gotten to the end without any non-zero exit codes...delete the mergeprogress file, we're g2g
     Cleanup-Incomplete-Progress $MountDirectory
 
-    return $pushedTags
+    return @($pushedTags)
 }
 
 $ErrorActionPreference = "Stop"
 
-# resolve the proxy location so that we can invoke it easily
+# this script requires the presence of git
+Test-Exe-In-Path -ExeToLookFor "git" | Out-Null
+
+# this script expects at least powershell 6 (core)
+
+if ($PSVersionTable["PSVersion"].Major -lt 6) {
```

Comment:
See my above comment, we can enforce this at the top of the script via:

```
#Requires -Version 6.0
#Requires -PSEdition Core
```

Reply:
❤️ TIL thanks @benbp !